ui/plugins/pluginpanel.html

```diff
@@ -210,6 +210,13 @@
           <label>Taskcluster:</label>
           <span><a href="{{line.url}}" target="_blank">{{line.value}}</a></span>
         </li>
+        <li class="small" ng-click="backfillJob()" ng-show="user.is_staff">
+          <label>
+            <a title="Trigger this jobtype back to the previous time it was run">
```

Comment:
Given that this functionality is not available to all the users bot only to the sheriffs I would keep using the term backfill in the title as well. Or come up with a simpler description, like "Trigger this job on previous result-sets."